pumps/sql.go

```diff
@@ -201,3 +216,92 @@ func (c *SQLPump) WriteData(ctx context.Context, data []interface{}) error {
 
 	return nil
 }
+
+func (c *SQLPump) WriteUptimeData(data []interface{}) {
+	c.log.Debug("Attempting to write ", len(data), " records...")
+
+	var typedData []analytics.UptimeReportData
+
+	batch := 500
+	newBatch := false
+	var firstDate time.Time
+	for i, r := range data {
```

Comment:
if `c.SQLConf.TableSharding` is `false`, what is the purpose of this `for`?

Reply:
we need to transform the data from `[]interface` to `[]analytics.UptimeReportData`,  decoding all the items on data.